### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by ubergeek42**

### CODE REVIEW

**Comment by ubergeek42** [comment]
@troy2914 followup to the previous fix ci PR

File: roles/balloonprinter/handlers/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-- name: restart gdm3
+- name: Restart gdm3
   ansible.builtin.service:
```
Comment (troy2914) [inline roles/balloonprinter/handlers/main.yml:3]: where is the corresponding task rename?

**Review by troy2914 — COMMENTED** [review]
just one nitpick.

File: roles/balloonprinter/handlers/main.yml (hunk 1 above)
Comment (ubergeek42) [reply roles/balloonprinter/handlers/main.yml:3]: ooh, good catch

Comment (ubergeek42) [reply roles/balloonprinter/handlers/main.yml:3]: fixed

**Review by troy2914 — APPROVED** [review]

**Pull request merged**